oak-security-spi/src/main/java/org/apache/jackrabbit/oak/spi/security/authorization/permission/Permissions.java

```diff
@@ -29,13 +29,13 @@
 import org.jetbrains.annotations.Nullable;
 
 import javax.jcr.Session;
-import java.util.Arrays;
 import java.util.HashSet;
 import java.util.Iterator;
 import java.util.LinkedHashMap;
 import java.util.Map;
 import java.util.Set;
-import java.util.stream.Collectors;
+
+import static java.util.stream.Collectors.toSet;
 
```

Comment:
We should agree on a consistent strategy whether the Collector imports should be static (I'm leaning towards "no")..